eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgPaymentsDb.scala

```diff
@@ -16,27 +16,26 @@
 
 package fr.acinq.eclair.db.pg
 
-import java.sql.ResultSet
-import java.util.UUID
-
 import fr.acinq.bitcoin.ByteVector32
 import fr.acinq.bitcoin.Crypto.PublicKey
 import fr.acinq.eclair.MilliSatoshi
 import fr.acinq.eclair.db.Monitoring.Metrics.withMetrics
-import fr.acinq.eclair.db.pg.PgUtils.DatabaseLock
-import fr.acinq.eclair.db._
+import fr.acinq.eclair.db.{HopSummary, _}
```

Comment:
nit:

```suggestion
import fr.acinq.eclair.db._
```

Reply:
How do you spot those? My import optimizer is totally oblivious.